### ISSUE TITLE
Improve "open-sourceness"

### ISSUE THREAD

**Issue by casassg** [HackAssistant/registration#154]
I would like to make this easier to fork and work for other hackathons. Any advice?

I was thinking of either moving this to my profile or creating a new organization just for this.

Also, even though I like how simple is it to say backend, maybe adding a name to it would make sense if other hackathons need to use it.

What do you think?

**To add**
- [x] Heroku deployment
- [ ] Docker deployment
- [ ] Add REST API

**Comment by dasilvacontin** [comment]
`hackathon-backend`. So it's just an API, no frontend?

**Comment by dasilvacontin** [comment]
Is there really other people interested in using this? What do those people due/use for now that this is not usable?

**Comment by dasilvacontin** [comment]
Maybe making it ready-to-use with Heroku, or with Docker.

**Comment by casassg** [comment]
@dasilvacontin There's no API actually, it's a full-stack application. It includes a small frontend for registration and more. Similar to what PennApps, MHacks and HackTheNorth currently have.

There are a few hackathon organizers that asked me about this. Afaik HackJunction thought on using it, we are using it for HackCU (I made a huge refactor there that I'll bring here as well), and other organizers asked me during HackUPC.

Also, thanks about the Heroku and Docker suggestions

**Comment by mvdan** [comment]
I'd say making it have APIs, or making the data easily accessible somehow, would be really important for this to be useful to other hackathons in general.

**Comment by casassg** [comment]
Adding the APIs to the todo list, should be easy with DRF. 

Even though what do you guys think about moving this to a different team or profile

**Comment by qaisjp** [comment]
agreed. this would be used by more hackathons if it was a standalone
project, rather than attached to a specific hackathon. giving it its own
name under the same org (hackupc) would still work (rather than just being
"backend").

**Comment by casassg** [comment]
The reason I was thinking on moving it away from hackupc org was to have a fork where we can have any specific hackupc stuff. Similar to what has been done with https://github.com/hackmit/quill

**Comment by casassg** [comment]
Moved to a new organization: github.com/hackAssistant and renamed to registration to make it more clear. This way I can transfer the maintainability to other hackathon organizers when I graduate.

**Comment by qaisjp** [comment]
I like this new hackassistant organisation.
